scaling_performance/optimizing_storage.adoc

```diff
@@ -12,6 +12,86 @@
 toc::[]
 
 == Optimizing Storage
+== General Storage Guidance
+
+The following is a matrix which details the persistent storage backend types {product-title} can be used with, and includes implementation examples.
+
+=== Available Storage Options
+[format="csv",width="70%",cols="3"]
+[options="header]
+|=======================
+Storage Type,Description,Examples
+"Block","This type of storage is presented to the operating system as a block device. It is suitable for applications that need full control of storage and operate at a low level on files bypassing the filesystem.  It is sometimes also referred to as Storage Area Network (SAN).  It is usually a non-shareable type of storage, which means that only one client at a time can mount an endpoint of this type.","CNS/CRS GlusterFS ^(1)^, iSCSI, Fibre Channel, Ceph RBD, OpenStack Cinder, Dell/EMC Scale.IO, AWS EBS ^(1)^, Dell/EMC Scale.IO, VMware vSphere Volume, GCE Persistent Disk ^(1)^, Azure Disk"
+File,"This type of storage is presented to the operating system as a file system export to be mounted.  File Storage is sometimes also referred to as Network Attached Storage (NAS).  Concurrency, latency, file locking mechanisms and other capabilities will vary widely between protocols, implementations, vendors, and scales.  Applications which natively operate on files may make use of this type of storage.","CNS/CRS GlusterFS ^(1)^, RHEL NFS, NetApp NFS ^(2)^, Azure File, Vendor NFS, Vendor GlusterFS ^(3)^, Loopback Image over NFS, OpenStack Manila, Azure File, AWS EFS"
+Object,"This type of is storage is accessible via a REST API endpoint.  Object Storage is configurable for use in the {product-title} Registry.  Applications intending to use this storage must build their drivers into the application and/or container.","CNS/CRS GlusterFS ^(1)^, Ceph Object Storage, OpenStack Swift, Aliyun OSS, AWS S3, Google Cloud Storage, Azure Blob Storage, Vendor S3 ^(3)^, Vendor Swift ^(3)^"
+|=======================
+NOTE: As of {product-title} 3.6.1, CNS GlusterFS (Container-Native Storage, a hyperconverged or cluster-hosted storage solution) and CRS GlusterFS (Container-Ready Storage, an externally hosted storage solution) provides interfaces for Block Storage, File Storage and Object Storage for the purpose of the {product-title} Registry, Logging, and Metrics.
```

Comment:
Again, are we certain CNS provides an interface to request Object Storage **natively**from openshift?

Reply:
Yup.